packages/core/src/messages/responses/cache-sorted-set-get-scores.ts

```diff
@@ -126,6 +133,16 @@ class _Hit extends Response {
     return this.valueRecordString();
   }
 
+  /**
+   * Returns the data as a Record whose keys and values are utf-8 strings, decoded from the underlying byte arrays.
+   * This can be used in most places where an Object is desired.  This is a convenience alias for
+   * {valueRecordStringString}.
+   * @returns {Record<string, number>}
+   */
+  public value(): Record<string, number> {
```

Comment:
This is the one semantically i'm not 100% sure about. Since the `get score` one has a `score` value, i wonder if `scores` is more appropriate here?

Reply:
hrm; yeah that's a fair callout. If we were starting over from scratch today I might be swayed to name the accessor on the `Hit` class `scores()`, but it's currently named `value*()`. So my response is basically the same one I gave to Nate, that I think that for consistency, these new accessors should have the same names as the existing ones.

We could definitely add `scores()` aliases to both the `Hit` class and the parent class here though in the future, if users call this one out as confusing!